webplugin/css/app/mck-sidebox-1.0.css

```diff
@@ -6462,6 +6462,28 @@ body.accesibility :focus {
 	-o-animation: km-slide-in-animation .5s ease .5s 1 forwards;
 	animation: km-slide-in-animation .5s ease .5s 1 forwards;
 }
+.mck-conversation-closed{
```

Comment:
@akshatdhabi can we change it to 'resolved'

Reply:
done